docs/rfc/rfc-19-random-beacon-v2.adoc

```diff
@@ -0,0 +1,157 @@
+:toc: macro
+
+= RFC 19: Random Beacon v2
+
+:icons: font
+:numbered:
+toc::[]
+
+== Background
+
+The last release of the random beacon got its genesis Sep 16th 2020 and worked
+until November 11th when it was stopped with a panic button as a result of an
+unannounced https://github.com/keep-network/keep-core/blob/main/docs/status-reports/2020-11-11-retro-geth-hardfork.adoc[Ethereum
+hard fork].
+
+For the time it was working, the beacon produced more than 1700 relay entries
+and 70 groups.
+
+Both on-chain and off-chain components proved to be successful but - just like
+with any other product - some pain points were identified by the users. This RFC
+aims at describing the pain points and proposing changes that should make the
+beacon easier to operate and more consistent with tBTC v2.
+
+=== ETH-based pricing
+
+Having reimbursements for various on-chain actions in ETH was a nice feature but
+also complex in practice. Highly volatile gas price led to creating new groups
+too often and gas price ceiling value was impossible to maintain up-to-date.
+With too many groups being created, operators were spending too much ETH on
+ticket submission and too low rewards were getting accumulated per group making
```

Comment:
can this hyperlnk to what ticket submission is?

Reply:
Unfortunately, we do not have an RFC describing it. I could link to https://docs.keep.network/random-beacon/index.html#_group_formation but this content will change for v2.